code/modules/awaymissions/zvis.dm

```diff
@@ -84,9 +84,14 @@
 
 /obj/effect/portal_sensor/proc/check_light()
```

Comment:
what even is this?

Reply:
I have no idea. I didn't want to fix it initially, but decided to anyway. It's in the hotel away mission, it seems.